setup-windows-sdk.ps1

```diff
@@ -25,6 +25,7 @@ cd "$sourceDir\build"
 
 Write-Output "Running qmake"
 $Env:PATH="C:\Program Files (x86)\CMake\bin;C:\Program Files\7-Zip;$Env:QT_BASE_DIR\bin;$Env:MINGW_BASE_DIR\bin;" + (($Env:PATH.Split(';') | Where-Object { $_ -ne 'C:\Program Files\Git\usr\bin' }) -join ';')
+[Environment]::SetEnvironmentVariable("QT_QPA_PLATFORM_PLUGIN_PATH", "$Env:QT_BASE_DIR\plugins\platforms", "User")
 qmake CONFIG+=debug ../src/mudlet.pro
```

Comment:
I don't want to be picky, but somehow I don't think a CMake based build will be expected to run **`qmake`** here!

😜

Reply:
That is existing code that is not related to cmake

Reply:
Huh, maybe not very fortunate place to put that in that case.
Cmake result needs that variable to know where to look for qwindows.dll